docker-compose.mail.yml

```diff
@@ -2,7 +2,7 @@ version: "3.5"
 
 services:
   mail:
-    image: tvial/docker-mailserver:release-v6.2.0
+    image: tvial/docker-mailserver:release-v7.0.0
```

Comment:
There seems to be a 7.0.1 already https://hub.docker.com/layers/tvial/docker-mailserver/release-v7.0.1/images/sha256-8d47cb0d2ecc31be1ffa42c5ae988ca61754099c3cc6cc1d8a5f45601e0c592d?context=explore

Reply:
That's correct. I'll update the image to 7.0.1.